Tasks/Task2/Task2.js

```diff
@@ -0,0 +1,112 @@
+﻿
+function isArray(arr) {
+    console.log("Is parameter a number?:\n");
+    console.log("Method 1:", Array.isArray(arr));   
+    console.log("Method 2:", arr.constructor == Array);
+    console.log("Method 3:", toString.call(arr) === "[object Array]");
```

Comment:
Please split to 3 methods, you should just return boolean value:
return Array.isArray(val);